### PR TITLE
Assignment 4 submission, ready for review

### PR DESCRIPTION
**Pull request by ralonsosirera**
Tasks 6 to 9, ready for review

### CODE REVIEW

File: Assignment4/ralonsosirera-190182/task06.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/ralonsosirera-190182/task06.py:12]: wont' work in python. Notebook magic

File: Assignment4/ralonsosirera-190182/task06.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith, VCARD.FN, Literal("Jane Smith")))
+g.add((ns.JaneSmith, VCARD.Given, Literal("Jane")))
+g.add((ns.JaneSmith, VCARD.Family, Literal("Smith")))
+# Visualize the results
+for s, p, o in g:
+    print(s,p,o)
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.worksAt, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/ralonsosirera-190182/task06.py:56]: property worksAt does not exist in vcard. Create your own

**Review by dgarijo — APPROVED** [review]
Small issues, but approved

File: Assignment4/ralonsosirera-190182/task06.py (hunk 2 above)
Comment (ralonsosirera) [reply Assignment4/ralonsosirera-190182/task06.py:56]: Hello, is it okay if I solve all the problems with !pip, properties and hierarchy in 7 by opening a new pr?

Comment (dgarijo) [reply Assignment4/ralonsosirera-190182/task06.py:56]: you may do as you prefer.

**Pull request merged**